ansible/roles/java-base/vars/main.yml

```diff
@@ -14,6 +14,7 @@ packages: {
   'freebsd': 'openjdk8-jre',
   'smartos': 'openjdk8',
   'ubuntu': 'openjdk-8-jre-headless',
+  'ubuntu1404': 'openjdk-7-jre-headless',
```

Comment:
No 8? Grr... Stubborn Ubuntu people

Reply:
No, not in the default package repositories. Let's handle this together in https://github.com/nodejs/build/issues/775